packages/widgets/src/media-transcoding/panels/statistics.panel.tsx

```diff
@@ -0,0 +1,139 @@
+import type react from "react";
+import type { MantineColor, RingProgressProps } from "@mantine/core";
+import { Box, Center, Grid, Group, RingProgress, Stack, Text, Title, useMantineColorScheme } from "@mantine/core";
+import { IconDatabaseHeart, IconFileDescription, IconHeartbeat, IconTransform } from "@tabler/icons-react";
+
+import { humanFileSize } from "@homarr/common";
+import type { TdarrPieSegment, TdarrStatistics } from "@homarr/integrations";
+import { useI18n } from "@homarr/translation/client";
+
+const PIE_COLORS: MantineColor[] = ["cyan", "grape", "gray", "orange", "pink"];
+
+interface StatisticsPanelProps {
+  statistics: TdarrStatistics;
+}
+
+export function StatisticsPanel(props: StatisticsPanelProps) {
+  const t = useI18n("widget.mediaTranscoding.panel.statistics");
+
+  const allLibs = props.statistics.pies.find((pie) => pie.libraryName === "All");
+
+  if (!allLibs) {
+    return (
+      <Center style={{ flex: "1" }}>
+        <Title order={3}>{t("empty")}</Title>
+      </Center>
+    );
+  }
+
+  return (
+    <Stack style={{ flex: "1" }} gap="xs">
+      <Group
+        style={{
+          flex: 1,
+        }}
+        justify="apart"
+        align="center"
+        wrap="nowrap"
+      >
+        <Stack align="center" gap={0}>
+          <RingProgress size={120} sections={toRingProgressSections(allLibs.transcodeStatus)} />
+          <Text size="xs">{t("transcodes")}</Text>
+        </Stack>
+        <Grid gutter="xs">
+          <Grid.Col span={6}>
+            <StatBox
+              icon={<IconTransform size={18} />}
+              label={t("transcodesCount", {
+                value: props.statistics.totalTranscodeCount,
+              })}
+            />
+          </Grid.Col>
+          <Grid.Col span={6}>
+            <StatBox
+              icon={<IconHeartbeat size={18} />}
+              label={t("healthChecksCount", {
+                value: props.statistics.totalHealthCheckCount,
+              })}
+            />
+          </Grid.Col>
+          <Grid.Col span={6}>
+            <StatBox
+              icon={<IconFileDescription size={18} />}
+              label={t("filesCount", {
+                value: props.statistics.totalFileCount,
+              })}
+            />
+          </Grid.Col>
+          <Grid.Col span={6}>
+            <StatBox
+              icon={<IconDatabaseHeart size={18} />}
+              label={t("savedSpace", {
+                value: humanFileSize(Math.floor(allLibs.savedSpace)),
+              })}
+            />
+          </Grid.Col>
+        </Grid>
+        <Stack align="center" gap={0}>
+          <RingProgress size={120} sections={toRingProgressSections(allLibs.healthCheckStatus)} />
+          <Text size="xs">{t("healthChecks")}</Text>
+        </Stack>
+      </Group>
+      <Group
+        style={{
+          flex: 1,
+        }}
+        justify="apart"
+        align="center"
+        wrap="nowrap"
```

Comment:
Seeing the screenshot I think it might just be missing the w="100%" here?
I know it's not really worth it but I feel like it's starring at me.